PyStemmusScope/save.py

```diff
@@ -0,0 +1,340 @@
+"""PyStemmusScope save module.
+
+Module designed to create a netcdf file following `ALMA
+convention <https://web.lmd.jussieu.fr/~polcher/ALMA/>`_ from csv files following
+`SCOPE format <https://scope-model.readthedocs.io/en/latest/outfiles.html>`_ in
+the output directory.
+
+The file
+`required_netcf_variables.csv <https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/main/utils/csv_to_nc/required_netcf_variables.csv>`_
+lists required variable names and their attributes based on `ALMA+CF
+convention table <https://docs.google.com/spreadsheets/d/1CA3aTvI9piXqRqO-3MGrsH1vW-Sd87D8iZXHGrqK42o/edit#gid=2085475627>`_.
+
+Example:
+
+    >>>from PyStemmusScope import StemmusScope
+    >>>from PyStemmusScope import save
+
+    >>>model = StemmusScope(config_file, exe_file)
+    >>>model.setup()
+    >>>model.run()
+    >>>save.to_netcdf(model.config, "required_netcf_variables.csv")
```

Comment:
```suggestion
    >>> from PyStemmusScope import StemmusScope
    >>> from PyStemmusScope import save

    >>> model = StemmusScope(config_file, exe_file)
    >>> model.setup()
    >>> model.run()
    >>> save.to_netcdf(model.config, "required_netcf_variables.csv")
```
You were missing spaces here.

Just fyi: you can preview the docs on your local machine by running;
`docs/make html`
The built docs will then be in `docs/build/html`, and you can see if the intended changes worked before committing a change.

Reply:
OK, this breaks the documentation because none of the `config_file`, `exe_file` and `required_netcf_variables.csv` exist! working on it.

Reply:
I replaced the example with a link to the notebook.

Reply:
Yes, that's the problem with actually running the examples... I think that is due to a plugin though, and we could choose to disable the checking of the docs as that makes no sense for this package.